ide/static/js/content.js

```diff
@@ -979,6 +980,34 @@ class Content extends React.Component {
     layer.info.phase = 0;
     this.setState({ net });
   }
+  saveModel(){
```

Comment:
As far as I understand this method performs update on already existing model so please rename it to update model

Reply:
This method doesn’t update on already existing model, it saves the model data on clicking the save button but if the model already exists in the database then it updates the model.

Reply:
So if you are doing this then why do we have older method still present?
 Why can't your remove that or add the code in same method?

Reply:
I can’t remover the older method as there is a difference between the 2 methods the older one is saving the model with shared properties while the new method is saving the model with not shared properties, if you still want me to combine the two, I’ll try to do it.